stdlib/builtin/thread.rbs

```diff
@@ -819,17 +820,18 @@ class Thread < Object
   #     }
   #     ...
   #     flag = false # stop thread
-  def self.pending_interrupt?: (*untyped args) -> bool
+  def self.pending_interrupt?: (?untyped error) -> bool
```

Comment:
`untyped` should be kind of `Exception` though https://github.com/ruby/ruby-signature/pull/271/files#r406779649